sklearn/exceptions.py

```diff
@@ -55,8 +55,6 @@ class ConvergenceWarning(UserWarning):
     >>> import numpy as np
     >>> import warnings
     >>> from sklearn.cluster import KMeans
-    >>> from sklearn.exceptions import ConvergenceWarning
-    >>> warnings.simplefilter("always", ConvergenceWarning)
```

Comment:
I don't know what to think about this.
Now the examples don't even use the objects they are supposed to illustrate?

Reply:
Yea, after thinking it over, this is a little weird. Maybe we should not have examples for custom exceptions or warnings.

Reply:
yeah I'd be fine with removing the examples